### DISCUSSION TITLE
Principiante HA: ho appena perso i comandi di Alexa!

### DISCUSSION THREAD

**Discussion by Ulisse87** [indomus/forum#1662] [GENERALE - Discussioni generiche]
Ciao a tutti.
Mi sono appena affacciato al mondo di HA e ho appena avuto una brutta notizia: se importi uno shelly su HA lo perdi come Alexa...perché? Non è possibile mantenere i comandi vocali di Alexa una volta integrata su HA?
E il bello che anche eliminando lo shelly da HA, non risulta comunque disponibile come comando vocale.
Perché

Grazie

**Comment by Scialla** [comment]
Sbagli approccio.
La corretta via è integrare shelly (e tutti gli altri device) in HA e non su alexa
poi da Ha esporre tutti i device integrati ad Alexa tramite haaska
